MAUI/Maps/migration.md

```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
+   </td>
+   <td>
+      {{'PrimaryValuePath'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[ShapeIDTableField](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeFileLayer.html#Syncfusion_SfMaps_XForms_ShapeFileLayer_ShapeIDTableField)'| markdownify }}
+   </td>
+   <td>
+      {{'ShapeDataField'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[Uri](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeFileLayer.html#Syncfusion_SfMaps_XForms_ShapeFileLayer_Uri)'| markdownify }}
+   </td>
+   <td>
+      {{'ShapesSource'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[ColorMappings](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html#Syncfusion_SfMaps_XForms_ShapeSetting_ColorMappings)'| markdownify }} in
+       {{'[ShapeSetting](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html)'| markdownify }} class
+   </td>
+   <td>
+      {{'ColorMappings'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[ShapeFill](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html#Syncfusion_SfMaps_XForms_ShapeSetting_ShapeFill)'| markdownify }} in
+       {{'[ShapeSetting](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html)'| markdownify }} class
+   </td>
+   <td>
+      {{'ShapeFill'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[ShapeStroke](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html#Syncfusion_SfMaps_XForms_ShapeSetting_ShapeStroke)'| markdownify }} in
+       {{'[ShapeSetting](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html)'| markdownify }} class
+   </td>
+   <td>
+      {{'ShapeStroke'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[ShapeStrokeThickness](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html#Syncfusion_SfMaps_XForms_ShapeSetting_ShapeStrokeThickness)'| markdownify }} in
+       {{'[ShapeSetting](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html)'| markdownify }} class
+   </td>
+   <td>
+      {{'ShapeStrokeThickness'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[ShapeColorValuePath](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html#Syncfusion_SfMaps_XForms_ShapeSetting_ShapeColorValuePath)'| markdownify }} and {{'[ShapeValuePath](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html#Syncfusion_SfMaps_XForms_ShapeSetting_ShapeValuePath)'| markdownify }} in
+       {{'[ShapeSetting](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html)'| markdownify }} class
+   </td>
+   <td>
+      {{'ShapeColorValuePath'| markdownify }}
+   </td>
+</tr>
+</table>
+
+The following code example explains how to intialize the shape sublayer and customize the layer in Xamarin SfMaps and .Net MAUI SfMaps.
```

Comment:
The following code example explains how to initialize the shape sublayer and customize the layer in the Xamarin SfMaps and .Net MAUI SfMaps.

Reply:
Modified